src/main/java/com/automation/school/ad_main/Collection/CompanyOrder.java

```diff
@@ -0,0 +1,87 @@
+package com.automation.school.ad_main.Collection;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyOrder implements Order{
+
+
+    private long id; //(generate automatically in constructor )
+    private LocalDateTime date; //(set as current day directly in constructor)
+    private User user;
+    public List<Company> companies;
+    private static int nextUniqueId = 0;
+    private int itemCount = 0;
+    Double value;
+
+    // public  User newUser;
+    //public  Item newItem;
+    //get the value of id
+    public CompanyOrder() {
+        this.date = LocalDateTime.now();
+        this.companies = new ArrayList<>(itemCount);
+        this.value = value;
+    }
+
+
+
+
```

Comment:
don't leave so much space between code, 1 line is enough

Reply:
Space between code were removed.